server/routes/postroutes.js

```diff
@@ -0,0 +1,10 @@
+import { Router } from 'express';
+import PostController from '../controllers';
```

Comment:
Your import style will be affected here when you change the export style in the index file.
You will import in this format: 
import { ... } from '...';

Reply:
Got it. Thanks for the review